.github/workflows/regression-tests.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
This gives a popup that says “Use workflow from,” unlike the UI for Integration tests: https://github.com/matter-labs/era-compiler-llvm/blob/def05be5e7e84b47ad703dba6e6c3e8262a5281d/.github/workflows/compiler-integration-tests.yml#L19, which gives separate choices for “compiler-llvm branch” and the workflow branch.  I don’t think I ever used that feature, always choosing the same branch for both.  If this change does that, then I think this is just a cosmetic issue, and we can just document internally that here “Use workflow from” also chooses  “compiler-llvm branch”